response.go

```diff
@@ -142,7 +142,7 @@ func (r *Response) ButtonTemplate(text string, buttons *[]StructuredMessageButto
 }
 
 // GenericTemplate is a message which allows for structural elements to be sent
-func (r *Response) GenericTemplate(text string, elements *[]StructuredMessageElement) error {
+func (r *Response) GenericTemplate(elements *[]StructuredMessageElement) error {
```

Comment:
I have no problem with this change, but it'd be great if you added a note about it in the [Breaking Changes](https://github.com/paked/messenger#breaking-changes) section of the README.